### PR TITLE
User login ui

### PR DESCRIPTION
**Pull request by NikolasGraham**

### CODE REVIEW

File: src/views/Login/Login.js [hunk 1]
```diff
@@ -5,27 +5,77 @@
...
+        }
+    };
+
+    //These settings hide and display developer dropdown menu on navbar
+    const [isOpenNav, setIsOpenNav] = useState(false);
+    const toggleNav = () => setIsOpenNav(!isOpenNav);
+
+    //These settings hide and display the alert
+    const [visibleIncorrectInformation, setVisibleIncorrectInformation] = useState(false);
+    const onDismiss = () => setVisibleIncorrectInformation(false);
```
Comment (blazinaj) [inline src/views/Login/Login.js:37]: Great job on the Alerts. This is good UX

File: src/hooks/useMongo.js [hunk 2]
```diff
@@ -15,17 +15,22 @@ export const useMongo = (input) => {
...
                 console.log(`successfully logged in with id: ${authedUser.id}`);
                 setAuthenticatedUser(authedUser);
                 setIsLoggedIn(true);
             })
-            .catch(err => console.error(`login failed with error: ${err}`))
+            .catch(err => {
+                console.error(`login failed with error: ${err}`);
+                error = err;
+            })
+        return error;
```
Comment (blazinaj) [inline src/hooks/useMongo.js:33]: Thank you for extending this to incorporate error handling.

**Review by blazinaj — APPROVED** [review]
This looks great, the UI is semantic and the state/function management is clean. Well done! 👍

File: src/views/Login/Login.js [hunk 3]
```diff
@@ -5,27 +5,77 @@
...
+                            </DropdownMenu>
+                        </UncontrolledDropdown>
+                    </Nav>
+                </Collapse>
+            </Navbar>
+            <Alert color="info" isOpen={visibleIncorrectInformation} toggle={onDismiss}>
+                Incorrect email or password!
+            </Alert>
+            <Form>
+                <div class="col-xs-9 col-md-7" id={"inputFieldsLogin"}>
```
Comment (toha87) [inline src/views/Login/Login.js:64]: Good job on this one! Bootstrap is cool!

**Review by toha87 — APPROVED** [review]
Good job Nik!

**Pull request merged**